Cargo.toml

```diff
@@ -226,4 +226,7 @@ substrate-build-script-utils = { git = "https://github.com/paritytech//substrate
 substrate-frame-rpc-system = { git = "https://github.com/paritytech//substrate.git", branch = "polkadot-v0.9.39" }
 substrate-prometheus-endpoint = { git = "https://github.com/paritytech//substrate.git", branch = "polkadot-v0.9.39" }
 substrate-state-trie-migration-rpc = { git = "https://github.com/paritytech//substrate.git", branch = "polkadot-v0.9.39" }
-substrate-wasm-builder = { git = "https://github.com/paritytech//substrate.git", branch = "polkadot-v0.9.39" }
\ No newline at end of file
+substrate-wasm-builder = { git = "https://github.com/paritytech//substrate.git", branch = "polkadot-v0.9.39" }
+
+[profile.release]
+overflow-checks = true
```

Comment:
do you want to enable this for everything? Or just for hyperspace?

Reply:
I think for now let's do for everything, it won't hurt